README.md

```diff
@@ -2,12 +2,21 @@
 
 Pantheon CMD is a Python-based command-line tool that allows you to generate a rendered preview of modular documentation using the new HAML templates.
 
+Installing the Pantheon CMD tool using RPM allows users to perform calls using the predefined aliases such as: 
```

Comment:
s/the Pantheon CMD tool/Pantheon CMD/
s/allows users to perform calls/allows you to perform actions/